Snowflake.Data/Client/SnowflakeDbConnection.cs

```diff
@@ -35,6 +37,14 @@ public class SnowflakeDbConnection : DbConnection
         // Will fix that in a separated PR though as it's a different issue
         private static Boolean _isArrayBindStageCreated;
 
+        private enum TransactionRollbackStatus
+        {
+            Undefined,

```

Comment:
do we need Undefined and NotNeeded status? If yes, please put comment when it's going to use.

Reply:
Right now just one, I will comment on the value